CONTRIBUTING.md

```diff
@@ -30,10 +30,9 @@ Check out your fork, switch to the newly created branch and start fixing the iss
 Here are the steps to get you started:
 
 * Make sure you have [NodeJS][] of v10 or higher installed.
-* Make sure you have [Yarn][] installed globally. You can use NPM instead, if you'd like.
-* Open your CLI in project's dir and run `yarn` (or `npm i`).
-* Run `yarn test-watch` (or `npm run test-watch`) to enable TDD.
-* Make sure to run `yarn lint` (or `npm run lint`) before making commit.
+* Open your CLI in project's dir and run `npm i`.
```

Comment:
I thought that yarn supports package-lock.json. We can add explanation for yarn too

Reply:
The support through import has it's down sides. I would not advertise this. People who are Yarn fans would know about this option anyway.